src/zrc/api/serializers.py

```diff
@@ -446,3 +537,110 @@ class Meta:
                 'lookup_field': 'uuid',
             },
         }
+
+
+class ResultaatSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Resultaat
+        fields = (
+            'url',
+            'zaak',
+            'resultaat_type',
+            'toelichting'
+        )
+        extra_kwargs = {
+            'url': {
+                'lookup_field': 'uuid',
+            },
+            'zaak': {
+                'lookup_field': 'uuid',
+            },
+            'resultaat_type': {
+                'validators': [
+                    # TODO: Add shape-validator when we know the shape.
+                    URLValidator(),
```

Comment:
I'm missing the `get_auth=get_ztc_auth` kwarg here
```suggestion
                    URLValidator(get_auth=get_ztc_auth),
```